packages/shared/lib/core/wallet/utils/buildNftOutputData.ts

```diff
@@ -0,0 +1,31 @@
+import { UnlockConditionTypes } from '@iota/types'
+import type { BuildNftOutputData } from '@iota/wallet'
+import { Converter } from '@core/utils'
+import { ADDRESS_TYPE_ED25519, EMPTY_HEX_ID, FEATURE_TYPE_METADATA, UNLOCK_CONDITION_ADDRESS } from '../constants'
+import { convertBech32ToHexAddress } from './convertBech32ToHexAddress'
+import { IIrc27Metadata } from '@core/nfts/interfaces'
+
+export function buildNftOutputData(metadata: IIrc27Metadata, address: string): BuildNftOutputData {
+    const unlockConditions: UnlockConditionTypes[] = [
+        {
+            type: UNLOCK_CONDITION_ADDRESS,
+            address: {
+                type: ADDRESS_TYPE_ED25519,
+                pubKeyHash: convertBech32ToHexAddress(address),
+            },
+        },
+    ]
+
+    const immutableFeatures = [
```

Comment:
Maybe type this as `FeatureType[]` for consistency instead of coercing the type to be of `BuildNftOutputData` on l.26?